sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -246,14 +255,14 @@ public virtual RecognizeCustomFormsOperation StartRecognizeCustomForms(string mo
         }
 
         /// <summary>
-        /// Recognizes pages from one or more forms, using a model trained without labels.
```

Comment:
Oh yes, good catch.  This method is now independent of whether the model was trained with or without labels.